src/commands/ExportAll.ts

```diff
@@ -124,11 +127,15 @@ export class ExportAll {
       // Check if there are still files after the filter
       if (filesToExport && filesToExport.length > 0) {
         let output = filesToExport.map((item) => {
-          const fileWithoutExtension =
-            item.type === "folder" ? item.name : path.parse(item.name).name;
-          return `export * from ${quote}./${fileWithoutExtension}${quote}${
-            semis ? ";" : ""
-          }\n`;
+          let filePath: string;
+          if (keepExtensions) {
+            filePath =
+              item.type === "folder" ? path.join(item.name, "index.ts") : item.name;
```

Comment:
There is this issue it will default to `index.ts` if it's importing a folder in the barrel, so perhaps somehow it needs to work for `.js` and other extensions. Not sure what is the right solution here, maybe a configuration or else check the file system and try all extensions until one succeeds. There might be a lib for that.